.travis.yml

```diff
@@ -55,10 +58,16 @@ install:
         $PIP install numpy==1.10.4 cython==0.24
         $PIP install -r tests/test_minimal_requirements.txt
       else
-        # HACK: install numpy and cython so cykdtree builds
-        $PIP install numpy==1.15.3 cython==0.29
+        # Getting cartopy installed requires getting cython and numpy installed
+        # first; this is potentially going to be fixed with the inclusion of
+        # pyproject.toml in cartopy.
+        # These versions are pinned, so we will need to update/remove them when
+        # the hack is no longer necessary.
+        $PIP install numpy==1.16.2 cython==0.29
         $PIP install -r tests/test_requirements.txt
+        $PIP install cartopy==0.17.0
       fi
+      pip install git+git://github.com/cykdtree/cykdtree.git
```

Comment:
I'd sugget
```suggestion
      $PIP install git+git://github.com/cykdtree/cykdtree.git
```

Reply:
Thanks for catching this, this was from me poking around trying to fix a weird issue I didn't understand. It ended up being caused by installing an old version of numpy over a new one. I've fixed that issue and backed out this change.